radarr/radarr.go

```diff
@@ -431,3 +431,22 @@ func (r *Radarr) SendCommand(cmd *CommandRequest) (*CommandResponse, error) {
 
 	return &output, nil
 }
+
+// Lookup will search for movies matching the specified search term.
+func (r *Radarr) Lookup(term string) ([]Movie, error) {
+	if term == "" {
+		return nil, nil
+	}
+
+	var out []Movie
+
+	params := make(url.Values)
+	params.Set("term", url.QueryEscape(term))
```

Comment:
`Encode()` runs `QueryEscape`, see [here](https://cs.opensource.google/go/go/+/refs/tags/go1.17.5:src/net/url/url.go;l=986). That means it'll run twice. The fix is to change this line to:
```suggestion
	params.Set("term", term)
```